### PR TITLE
Added a footer to main website

### PR DESCRIPTION
**Pull request by Ellusionists**
https://ellusionists.github.io/opencodecollab/  -> gh-page to view changes made by me.

### CODE REVIEW

**Comment by gauthamzz** [comment]
Im sorry you need to syc your fork with master and then  make a branch and send pr

**Comment by CosmicCoder96** [comment]
@Ellusionists It seems that you have manually copied some of the files without initialising a git repository beforehand or cloning it properly and then sent a pr to merge, which has resulted in your commits covering all of the files, which didn't even had any change to begin with. In other words you are not in sync with the main repository.

The correct way would be to fork it , clone your fork , add the main repository as an upstream (google , if any step seems ambiguous) 
Make the changes on your fork.
Make sure to add only those files which you have changes using git add filename .. and not all of the unnecessary changes.
then commit and push them on your fork.
Now you can make a pull request.

File: _data/students.yml [hunk 1]
```diff
@@ -108,3 +108,10 @@
   facebook: https://www.facebook.com/saicharan.reddy.927
   bio: New to github!
 
+
+- name: Pradeep Gangwar
```
Comment (gauthamzz) [inline _data/students.yml]: i dont think  you need to edit 53 files , you are doing this wrong , please make a new branch and send a pr 

please syc with master also

Comment (Ellusionists) [inline _data/students.yml]: I don't know what I'm doing wrong. I'm following the guidelines given on github help pages.. Doesn't seem to work...

Comment (gauthamzz) [inline _data/students.yml]: text me , ill help you out

**Pull request closed without merging**